rfcs/text/0011-sip-fields.md

```diff
@@ -129,11 +218,16 @@ Stage 1: Describe at a high-level how these field changes will be used in practi
 -->
 Typical implementations will utilize these fields to describe and normalize the various stages of a SIP/SDP based communcations mechanism.  Additional considerations including call analytics, fraud detection, troubleshooting, and threat detection have been identified as additional considerations.
 
+Related nesting changes:
+ - [user fields](text/0011/user.yml)
```

Comment:
Problem with using the top level fields is that a single sip message from the right spot in a conversation can contain multiple URLs and multiple users.

Reply:
on the scope of fields, I'm doubting that zeek/packetbeat represent a useful line in the sand for usefulness as the bulk of real world SIP traffic will be encrypted, based on the feedback from the gentleman looking at parsing core sip logs from various sip apps, I tried to cover the use cases that seemed most relevant. TBH I'm not sure which items could be removed while maintaining a basic coverage for a real world integration.

Reply:
I'm still unsure about moving forward with adding 140+ new fields (looking in the `master` branch today, ECS is at 859 total fields, including field reuses) just for SIP/SDP. Relative to what's been captured in other ECS network protocol-focused field sets, that's pretty significant. 

I'm concerned we've focused too much on exhaustively capturing details of the protocols instead of focusing on a smaller list of fields to capture common SIP logs and/or metrics within ECS.